### ISSUE TITLE
Image of full body in trainingset

### ISSUE THREAD

**Issue by buehlpa** [zalandoresearch/fashion-mnist#184]
Hi, 
I just wanted to report an image where an entire person is shown with a shirt and trousers, so different from the other images. 
It's image 36325 in the training set with class: shirt.

I've read #166 and already saw your answers there, and just thought might be interesting to know :) 

`
from tensorflow.keras.datasets import fashion_mnist
import matplotlib.pyplot as plt
(x_train, y_train), _ = fashion_mnist.load_data()
plt.imshow(x_train[36325], cmap='gray') 
plt.title(f'TF, label:{y_train[36325]}, idx: 36325')
plt.show()
`
![image](https://github.com/zalandoresearch/fashion-mnist/assets/64488738/127600e0-3a05-4b9d-974a-2142900c9412)

Cheers Pascal

**Comment by buehlpa** [comment]
some more :D

![image](https://github.com/zalandoresearch/fashion-mnist/assets/64488738/ca64d8af-aa5a-4e3a-9b13-721d945d806f)
